Use regex from C++11 instead of boost/tr1's version (the latter is gone as of
boost 1.65).
--- configure.orig	2015-11-24 13:45:55 UTC
+++ configure
@@ -17554,13 +17554,13 @@ cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
 
-#include <boost/tr1/regex.hpp>
+#include <regex>
 
 int
 main ()
 {
-  std::tr1::regex r ("te.t", std::tr1::regex_constants::ECMAScript);
-  return std::tr1::regex_match ("test", r) ? 0 : 1;
+  std::regex r ("te.t", std::regex_constants::ECMAScript);
+  return std::regex_match ("test", r) ? 0 : 1;
 }
 
 _ACEOF
@@ -17631,13 +17631,13 @@ cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
 
-#include <boost/tr1/regex.hpp>
+#include <regex>
 
 int
 main ()
 {
-  std::tr1::regex r ("te.t", std::tr1::regex_constants::ECMAScript);
-  return std::tr1::regex_match ("test", r) ? 0 : 1;
+  std::regex r ("te.t", std::regex_constants::ECMAScript);
+  return std::regex_match ("test", r) ? 0 : 1;
 }
 
 _ACEOF
